src/validators.ts

```diff
@@ -224,3 +224,25 @@ export function max(maximum: number, message?: string): Validator {
     execute: executeMax(maximum)
   }
 }
+
+const WORD_REGEX = /^[a-zA-Z]+$/
```

Comment:
As a slight modification, let's go with the expression `/^[a-zA-Z]*$/` so that it accepts empty string.  The `required` validator has responsibility for protecting against empty strings.